src/cuter_analyzer.erl

```diff
@@ -199,3 +201,27 @@ calculate_coverage(Feasible, Visited) ->
       Coverage = 100 * (All - NotVisited) / All,
       {All - NotVisited, All, Coverage}
   end.
+
+%% ----------------------------------------------------------------------------
+%% Report collected metrics.
+%% ----------------------------------------------------------------------------
+
+-spec report_metrics() -> ok.
+report_metrics() ->
+  case cuter_metrics:get_distribution_metrics() of
+    [] -> ok;
+    Names ->
+      cuter_pp:pp_metrics_title(),
+      report_distribution_metrics(Names)
+  end.
+
+report_distribution_metrics([]) ->
```

Comment:
For uniformity with the function above, the `ok`s should not be in a separate line in this function.

Reply:
Done..